x.py

```diff
@@ -18,6 +18,10 @@
     DebianArch("i386", "i386", "i686-unknown-linux-gnu"),
 ]
 
+debian_non_buster_arches = [
+    DebianArch("ppc64le", "ppc64el", "powerpc64le-unknown-linux-gnu"),
```

Comment:
These two are meant to be different correct?

Reply:
Yes, that is correct. The value for bashbrew is ppc64le while the dpkg architecture value needs to be ppc64el.